spam/02_spam_data_augmentation_tutorial.py

```diff
@@ -35,10 +35,13 @@
 # For more details, check out the labeling functions [tutorial](https://github.com/snorkel-team/snorkel-tutorials/blob/master/spam/spam_tutorial.ipynb).
 
 # %%
+import numpy as np
 import os
+import random
 
 # For reproducibility
-os.environ["PYTHONHASHSEED"] = "0"
```

Comment:
We don't need this? Seems necessary for Keras from everything I've read.

Reply:
It didn't seem to make a difference. But I'll add it back

Reply:
I'm guessing this is needed if you use the regular python hash, but we are using md5, which is deterministic. But I'll keep it in for future robustness.